packages/cli/package.json

```diff
@@ -24,7 +24,7 @@
 	"devDependencies": {
 		"@clack/core": "^0.3.2",
 		"@clack/prompts": "^0.6.3",
-		"@cloudflare/eslint-config-worker": "*",
+		"@cloudflare/eslint-config-worker": "workspace:*",
```

Comment:
I ran into some issues with pnpm trying to resolve these from npm.

Does anyone have any idea what I can do here? I can remove these if its not desired

Reply:
I'm not sure why you needed to do these changes. They have not been needed previously. Is there something funny in your local setup perhaps?
Arguably putting workspace makes these dependencies tighter so they will only work if the package matches within the workspace. But given that in both cases we have `*` then both should match anything in the workspace.

Reply:
There is definitely something funny in my local setup, totally happy to put these back.

I couldn't get it to resolve them without it. There are some places that already had `workspace:*`

There shouldn't be anything too funny with my setup. I'm using volta to install the node.js version required in the package.json and the latest pnpm.